docs/usage.md

```diff
@@ -12,7 +12,7 @@ Commands:
   help        Print this message or the help of the given subcommand(s)
 
 Options:
-      --debug...  Turn debugging information on
+      --debug...  Turn debugging information on. Use twice (--debug --debug) for trace-level logs.
```

Comment:
Seems weird. Why not have --trace?

Reply:
yeah, I was following the intended format, 

https://github.com/open-telemetry/weaver/blob/7beff49c71cf2d17a04071b72386138fcfabd2b6/src/cli.rs#L21-L22

and was wondering if there is a convention. I realized it's similar to what curl does with `-v`, `-vv`, `-vvv`. Curious how @lquerel envisioned it.

Reply:
The intention was to be able to define the verbosity level. It’s common practice to have `-v`, `-vv`, `-vvv`, but I didn’t specify that I wanted the short `-v` and not `-d`. Sorry for this oversight. If you prefer to use `--trace`, I have no objection.